notebooks/05_operationalize/aks_locust_load_test.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>I think this is expanding the cells when the notebook is running, but not when this is viewed on Github. In other notebooks we have all the imports on the first cell, I think we should put this in the beginning as well</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
yup, will do